Actions/AL-Go-Helper.ps1

```diff
@@ -269,59 +291,95 @@ function DownloadAndImportBcContainerHelper {
         $bcContainerHelperVersion = "https://github.com/$owner/navcontainerhelper/archive/master.zip"
     }
 
-    if ($bcContainerHelperVersion -eq "none") {
-        $tempName = ""
+    if ("$env:BcContainerHelperPath" -and (Test-Path -Path $env:BcContainerHelperPath -PathType Leaf)) {
```

Comment:
Settings for bcContainerHelper and the BcContainerHelper version number is not supported in GitHub repo settings or org settings variables. Will add that to the settings doc.

Reply:
Have moved the check up.

Reply:
I still don't get why we do that custom "read settings".
What if the settings are more than 2 levels up?

Why not call `ReadSettings` to get that version? Now that `$env:BcContainerHelperPath` is set after the first action that calls `DownloadAndImportBcContainerHelper`, is there a problem to not do it?

Reply:
I think that in this case I actually do not want to have any defaulting or other things applied to the settings.

I will investigate.

Reply:
We do not want to apply organizational variables or project settings here - meaning that a call to ReadSettings should specifically set a number of parameters to [string]::empty in order to avoid that.
That is very confusing.

Reply:
Why is `bcContainerHelperVersion` a special case setting? Why can't it be a project setting or an org variable?

You mentioned adding it to the settings doc, maybe you forgot?

Reply:
The ContainerHelperVersion is really defined by AL-Go-Actions - if AL-Go-Actions require a later version of ContainerHelper, then AL-Go-Actions will define that in AL-Go-Helper.
The few cases where it makes sense to override the containerhelper version in settings are for troubleshooting or workarounds.
Example (real life). partner is using latest AL-Go (using latest ContainerHelper), next version of BC ships with a bug that affects partner workflows. Workaround is added to ContainerHelper for this issue and partner specifies preview in settings.
So, it is really not a setting people should use and it is definitely not something people should put in org settings.